pages/dao/[symbol]/proposal/components/instructions/raydium/AddLiquidity.tsx

```diff
@@ -85,10 +85,10 @@ const AddLiquidityRaydium = ({
   const programId: PublicKey | undefined = realmInfo?.programId
   const [form, setForm] = useState<AddLiquidityRaydiumForm>({
     governedAccount: undefined,
-    baseTokenName: '',
-    quoteTokenName: '',
-    baseAmountIn: 0,
-    quoteAmountIn: 0,
+    tokenAName: '',
```

Comment:
Could replace A by base and B by quote everywhere

Reply:
I wanted to do it only in the form because it does not matter for the raydium instruction at this level. But Raydium' pool actually defines UXP as the base and USDC as the quote, it is not interchangeable. The UI form allows to switch tokenA and tokenB in order to make it easier for users to know which amount they want to deposit.

Reply:
Ok, but it could allow to switch token Base and token Quote too 😅
But it's just nit, lgtm